### DISCUSSION TITLE
How to know who is leader about ceph mon through resources of rook?

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#11803] [Q&A]
Not through "ceph mon stat",  only through rook resources

**Comment by travisn** [comment]
Rook resources don't have that status. Just curious, what is the scenario? That's a ceph detail that doesn't seem necessary for rook users.

If it helps, we could add a new command to the [krew plugin](https://github.com/rook/kubectl-rook-ceph) to get info like this.

**Reply by zhucan** [reply]
if we can know who is leader from rook resources, we need to check who is leader from ceph cmd.
